jib-core/src/main/java/com/google/cloud/tools/jib/docker/CliDockerClient.java

```diff
@@ -99,21 +99,41 @@ public List<DescriptorDigest> getDiffIds() throws DigestException {
     }
   }
 
-  /** Default path to the docker executable. */
-  public static final Path DEFAULT_DOCKER_CLIENT = Paths.get("docker");
+  /** Default paths to the docker or alternatives executables. */
+  public static final List<Path> DEFAULT_DOCKER_CLIENT = Arrays.asList(
```

Comment:
I found the CONTAINER_ENGINES name better reflecting the purpose of the variable.

Reply:
Docker name is widely hardcoded everywhere, so I keep naming convention. Otherwise I should replace every "docker" word to container.